lib/maintenance_tasks/engine.rb

```diff
@@ -21,6 +21,14 @@ class Engine < ::Rails::Engine
       MaintenanceTasks.backtrace_cleaner = Rails.backtrace_cleaner
     end
 
+    initializer "maintenance_tasks.assets.precompile" do |app|
+      app.config.assets.paths << Engine.root.join("vendor/assets/stylesheets")
+      app.config.assets.paths << Engine.root.join("app/assets/stylesheets")
+      app.config.assets.paths << Engine.root.join("app/assets/javascripts")
```

Comment:
This is already done by the assets pipeline gem, or did you find something that broke because of it?